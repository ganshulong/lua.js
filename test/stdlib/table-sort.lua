local list = { 3, 2, 54, 2, 51 }
table.sort(list, function(a, b)
    return a < b
end)
print(table.unpack(list))
