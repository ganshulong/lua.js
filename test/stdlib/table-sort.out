2 2 3 51 54
